MdeModulePkg/Library/SmmPerformanceLib/SmmPerformanceLib.inf

```diff
@@ -18,7 +22,7 @@
   FILE_GUID                      = 1EDD13E6-D0CD-4432-A692-FF65C9B4F039
   MODULE_TYPE                    = DXE_SMM_DRIVER
   VERSION_STRING                 = 1.0
-  LIBRARY_CLASS                  = PerformanceLib|DXE_SMM_DRIVER
+  LIBRARY_CLASS                  = PerformanceLib|DXE_SMM_DRIVER MM_STANDALONE

```

Comment:
why add MM_STANDALONE in MdeModulePkg/Library/SmmPerformanceLib/SmmPerformanceLib.inf since you have added the MdeModulePkg/Library/SmmPerformanceLib/StandaloneMmPerformanceLib.inf?

Reply:
I believe that was left over from earlier code that shared a common implementation before the constructors and destructors were added and the implementation split. I removed it.